### PR TITLE
Filter ANSI escape codes for Sentry

### PR DESCRIPTION
**Pull request by jmhobbs**
Removes ANSI escape codes from error and console log messages before being sent to Sentry.

Additionally, using `environment` for  the `cli`/`action` distinction was incorrect.  Move that to `entrypoint` and set a build time variable for `SENTRY_ENVIRONMENT` to bake it in appropriately.

<details>
  <summary>📦 Published PR as canary version: <code>11.11.1--canary.1068.11147048454.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install chromatic@11.11.1--canary.1068.11147048454.0
  # or 
  yarn add chromatic@11.11.1--canary.1068.11147048454.0
  ```
</details>

### CODE REVIEW

File: node-src/errorMonitoring.ts [hunk 1]
```diff
@@ -1,15 +1,62 @@
 import * as Sentry from '@sentry/node';
+import stripAnsi from 'strip-ansi';
+
+/**
+ * Remove ANSI escape codes from Sentry ErrorEvents.
+ *
+ * @param event An event containing a captured exception.
```
Comment (jwir3) [inline node-src/errorMonitoring.ts:7]:
```suggestion
 * @param event A {@link Sentry.ErrorEvent} containing a captured exception.
```

File: node-src/errorMonitoring.ts [hunk 2]
```diff
@@ -1,15 +1,62 @@
...
+ *
+ * @returns The modified event.
```
Comment (jwir3) [inline node-src/errorMonitoring.ts:9]:
```suggestion
 * @returns The same {@link Sentry.ErrorEvent}, but modified to remove ANSI escape codes.
```

**Review by jwir3 — APPROVED** [review]
LGTM. Nice work adding a test to make sure that `stripAnsi` does what we expect!

File: node-src/errorMonitoring.ts [hunk 3]
```diff
@@ -1,15 +1,62 @@
...
+/**
+ * Remove ANSI escape codes from Sentry ErrorEvents.
+ *
+ * @param event A {@link Sentry.ErrorEvent} containing a captured exception.
+ *
+ * @returns The same {@link Sentry.ErrorEvent}, but modified to remove ANSI escape codes.
+ */
+export function filterErrorEvent(event: Sentry.ErrorEvent) {
+  // Remove ANSI escape codes from error messages
+  if (event?.message) {
```
Comment (codykaup) [inline node-src/errorMonitoring.ts:13]: Can `event` be undefined?

**Review by codykaup — APPROVED** [review]

File: node-src/errorMonitoring.ts (hunk 1 above)
Comment (codykaup) [reply node-src/errorMonitoring.ts]: Should we add those to the breadcrumb function too?

**Pull request merged**